assets/app/scripts/services/deployments.js

```diff
@@ -192,14 +192,18 @@ angular.module("openshiftConsole")
       );
     };
 
-    //deploymentConfigs is optional
-    DeploymentsService.prototype.associateDeploymentsToDeploymentConfig = function(deployments, deploymentConfigs) {
+    // deploymentConfigs is optional
+    // filter will run the current label filter against any deployments whose DC is deleted, or any RCs
```

Comment:
I'm not in love with putting this in here.  But otherwise there would have been a bunch of logic added to the controller duplicating loops happening here.  Filtering these out made other view logic much much simpler.